src/transactions/types.ts

```diff
@@ -100,7 +101,12 @@ export enum TransactionStatus {
   Failed = 'Failed',
 }
 
-export type TokenTransaction = TokenTransfer | TokenExchange | NftTransfer | TokenApproval
+export type TokenTransaction =
+  | TokenTransfer
+  | TokenExchange
+  | NftTransfer
+  | TokenApproval
+  | JumpstartDeposit
```

Comment:
```suggestion
  | JumpstartTransfer
```

nit: naming is hard, but there is a type called `JumpstartDeposit` in `TokenTransactionTypeV2` already, can we avoid duplicates? the "deposit" part in `TokenTransactionTypeV2` refers to a user putting funds in the jumpstart contract right...so then this transfer type should not also be called "deposit" since it also means users reclaiming and receiving funds from the jumpstart contract?

Reply:
This is related to the other comment too, I see how this is confusing.

In my proposed solution:
The received and reclaim are a new type of `TokenTransferV3`. At the end, it's just really a transfer that we want some special UX. And the new transaction type is the deposit that includes the relevant fields that are needed to execute the reclaim.

An alternative solution would be to put all the jumpstart transactions under a single `__typename` but the fact that the schema is different was taking me back from this solution.